manager/channels.go

```diff
@@ -24,8 +24,8 @@ type ChannelRepository interface {
 	// by the specified user.
 	One(string, string) (Channel, error)
 
-	// All retrieves the channels owned by the specified user.
-	All(string) []Channel
+	// Bulk retrieves the bulk of channels owned by the specified user.
+	Bulk(string, int, int) []Channel
```

Comment:
@mijicd What do you think of changing offset and limit to uint32 or uint64, according to theirs nature?
Also, what do you think of an idea of wrapping pagination result in pagination object. Something like:
```
type Page struct {
	offset uint32
	limit  uint32
	total  uint32
	data   interface{}
}
```

Reply:
Type choice is correct, though I prefer to use less specific variant. Not sure how would this change affect the signatures (repository & manager). In generally, it's not a bad idea, though some clarification is needed.

Reply:
I'm not sure yet, but it should not be problem to Manager or Repository. Limit and offset are already passed as params and data interface is, basically, list of... something, whatever. So, only `total` is questionable since I will call GORM's `Count` method inside repository method call. Not sure how that would affect performance. Response validation should be straightforward, I don't see any problems right now.

Reply:
I'm talking about request validation, we don't do any response validations. Regarding the second point, I don't recommend to avoid `Page` but return a tuple (array + count) as return value. Executing count will affect performance, but it can be handled, and these endpoints are not time critical anyhow.